dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/FeatureUtility.java

```diff
@@ -622,14 +622,19 @@ public void downloadFeatures(List<String> resolvedFeatures) throws InstallExcept
     }
 
     /**
-     * Extracts the feature name and version from an ESA filepath. Example:
-     * extractFeature(appSecurity-3.0-19.0.0.8.esa) returns appSecurity-3.0
-     * 
-     * @param filename
-     * @return
-     */
+	 * Extracts the feature name from an ESA filepath. Example:
+	 * extractFeature(appSecurity-3.0-19.0.0.8.esa) returns appSecurity-3.0
+	 * 
+	 * @param filename
+	 * @return
+	 */
     private String extractFeature(String filename) {
-        return filename.replaceFirst("(-\\d\\d\\.\\d\\.\\d\\.\\d\\.esa)", "");
+		String[] split = filename.split("-");
+		if (split.length > 2) {
```

Comment:
should this be `>=2` or `>1`? I'm thinking of a test string like `"appSecurity-3.0.19.0.0.8.esa"` which would return `appSecurity`.